courses/migrations/0016_enrollment_order.py

```diff
@@ -0,0 +1,76 @@
+# Generated by Django 2.1.7 on 2019-06-25 18:21
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def backpopulate_order(apps, schema_editor):
+    """
+    Set the order for existing enrollments.
+    """
+    CourseRunEnrollment = apps.get_model("courses", "CourseRunEnrollment")
+    ProgramEnrollment = apps.get_model("courses", "ProgramEnrollment")
+    Order = apps.get_model("ecommerce", "Order")
+    Product = apps.get_model("ecommerce", "Product")
+
+    def get_order(user, product):
+        """Return the order for a user and product if any"""
+        return (
+            Order.objects.filter(lines__product_version__product=product)
+            .filter(purchaser=user)
+            .order_by("id")
+            .first()
+        )
+
+    run_enrollments = CourseRunEnrollment.objects.filter(order=None).order_by("id")
```

Comment:
Out of paranoia, maybe make this and the `ProgramEnrollment` query below `iterator()`?

Reply:
Added `.iterator()`